my_view_inc.php

```diff
@@ -74,40 +73,17 @@
 $c_filter['assigned'] = filter_create_assigned_to_unresolved( helper_get_current_project(), $t_current_user_id );
 $t_url_link_parameters['assigned'] = FILTER_PROPERTY_HANDLER_ID . '=' . $t_current_user_id . '&' . FILTER_PROPERTY_HIDE_STATUS . '=' . $t_bug_resolved_status_threshold;
 
-$c_filter['recent_mod'] = array(
-	FILTER_PROPERTY_CATEGORY_ID => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_SEVERITY => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_STATUS => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_HIGHLIGHT_CHANGED => $t_default_show_changed,
-	FILTER_PROPERTY_REPORTER_ID => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_HANDLER_ID => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_RESOLUTION => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_BUILD => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_VERSION => array(
-		'0' => META_FILTER_ANY,
-	),
-	FILTER_PROPERTY_HIDE_STATUS => array(
-		'0' => META_FILTER_NONE,
-	),
-	FILTER_PROPERTY_MONITOR_USER_ID => array(
-		'0' => META_FILTER_ANY,
-	),
-);
-$t_url_link_parameters['recent_mod'] = FILTER_PROPERTY_HIDE_STATUS . '=none';
+# @TODO cproensa: make this value configurable
```

Comment:
You are better off adding an issue to track making "recent days" configuration than a code TODO.